### DISCUSSION TITLE
[Android] Where is the Maui.SplashTheme?

### DISCUSSION THREAD

**Discussion by AndreasReitberger** [dotnet/maui#8303] [Q&A]
I'm currently having a white ActionBar background, while my Splash backgroundcolor is blue.
So I created a "styles.xml" file on my own and did something like this.

```xml
	<style name="SplashTheme" parent="@style/Maui.SplashTheme">
		<item name="android:windowNoTitle">true</item>
		<item name="android:windowFullscreen">true</item>
		<item name="android:windowContentOverlay">@null</item>
		<item name="android:windowActionBar">true</item>
	</style>
```

In the `MainActivity.cs` file (Platforms/Android)
```cs
[Activity(
    //Theme = "@style/Maui.SplashTheme",
    Theme = "@style/SplashTheme",
    MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize)]
public class MainActivity : MauiAppCompatActivity
```

It seems to work, however I'm not sure if this is the accurate approach?
Thanks for your thoughts!

**Comment by davidjackson148** [comment]
where does that styles,.xml file live, what folder please?

**Comment by ewerspej** [comment]
Do you mean this? https://github.com/dotnet/maui/blob/main/src/Core/src/Platform/Android/Resources/values/styles.xml

**Comment by juniorsaraviao** [comment]
I'm updating the SplashTheme using the code above but I'm getting the error: `style/SplashTheme not found` on .Net MAUI 8.0.3. Any idea about that?

**Reply by ghost** [reply]
I dont undestand why but its helps for me
Resolved by Build -> 'Clean Solution', followed by 'ReBuild'.



https://stackoverflow.com/a/76578025

**Reply by PabloRuiz-TT** [reply]
Esto me funciono
